packages/govuk-frontend-review/src/views/full-page-examples/index.njk

```diff
@@ -21,21 +21,29 @@
     <hr class="govuk-section-break govuk-section-break--l" />
 
     {% for example in fullPageExamples %}
+    <section aria-labelledby="heading-{{ example.name }}">
+      <h2 id="heading-{{ example.name }}" class="govuk-heading-l govuk-!-margin-bottom-2">{{ example.name | unslugify }}</h2>
 
-      <h2 class="govuk-heading-m"><a href="/full-page-examples/{{ example.path }}" class="govuk-link">{{ example.name | unslugify }}</a></h2>
+      <p class="govuk-body">
+        <a href="/full-page-examples/{{ example.path }}" target="_blank" class="govuk-link">
```

Comment:
Worth adding `rel="noopener noreferrer"`?

Reply:
@owenatgov Yeah course, will do

Just a heads up, I copied it from here where we forgot:

https://github.com/alphagov/govuk-design-system/blob/7980f36135b37f8bfb73dd949dc397948f0fd8e2/views/partials/_example.njk#L33

Reply:
Oooo good spot. If I get a sec I'll patch that up as well.